ix-dev/community/jdownloader/templates/docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set reserved_args = [] %}
+{% if values.jdownloader.additional_args != values.jdownloader.additional_args|unique|list %}
+  {% do tpl.funcs.fail("JDownloader [jdownloader.additional_args] contains duplicate values [%s], please remove them" | format(values.jdownloader.additional_args|join(', '))) %}
+{% endif %}
+{% for arg in values.jdownloader.additional_args %}
+  {% if arg in reserved_args %}
+    {% do tpl.funcs.fail("JDownloader already inserts [%s] as an argument, please remove it from [jdownloader.additional_args]" | format(arg)) %}
+  {% endif %}
+{% endfor %}
+
+{% set c1 = tpl.add_container(values.consts.jdownloader_container_name, "image") %}
+{% set perm_container = tpl.deps.perms(values.consts.perms_container_name) %}
+{% set perms_config = {"uid": values.run_as.user, "gid": values.run_as.group, "mode": "check"} %}
+
+{% do c1.set_user(values.run_as.user, values.run_as.group) %}
+{% do c1.healthcheck.disable() %}
+{% set cmd = namespace(x=[]) %}
+{% if values.ci %}
+  {% do cmd.x.append("--hello-world") %}
+{% else %}
+  {% do cmd.x.extend(values.jdownloader.additional_args) %}
+  {% do cmd.x.append("run") %}
+{% endif %}
+
+{% do c1.set_command(cmd.x) %}
```

Comment:
From a quick look on the upstream image, overriding command will make the container not start. as the image uses command instead of entrypoint.